### PR TITLE
DOCS-1196: Modular resources review

### PR DESCRIPTION
**Pull request by sguequierre**
Just some small changes, couldn't find a lot to improve on with how throughly everyone has reviewed it already!

### CODE REVIEW

File: docs/modular-resources/_index.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ To create a new modular resource:
 1. Package your modular resource or modular resources as a {{< glossary_tooltip term_id="module" text="module" >}} and [upload the module to the Viam registry](/modular-resources/upload/) to make it available for deployment to robots.
    You can upload _private_ modules for your [organization](/manage/fleet/organizations/) or _public_ modules.
 1. Once you have uploaded your module to the registry, you can [deploy and configure the module](/modular-resources/configure/) from [the Viam app](https://app.viam.com/).
-   If the added resource is a component, you can then test it using the [Control tab](/manage/fleet/#remote-control) and [program](/program/) it with Viam's Go or Python SDKs.
+   If the added resource is a component or service, you can then test it using the [Control tab](/manage/fleet/#remote-control) and [program](/program/) it with Viam's Go or Python SDKs.
```
Comment (sguequierre) [inline docs/modular-resources/_index.md:72]: @andf-viam I was assuming that you'll now be able to program and test modular services like components, if I'm assuming incorrectly and this is just a quirk I'm unaware of LMK

File: docs/modular-resources/create/_index.md [hunk 2]
```diff
@@ -18,11 +18,11 @@ aliases:
   - "/extend/modular-resources/create/"
 ---
 
-You can extend Viam by creating a custom {{< glossary_tooltip term_id="module" text="module" >}} that provides one or more modular {{< glossary_tooltip term_id="resource" text="resources" >}} ([components](/components/) and [services](/services/)) or {{< glossary_tooltip term_id="model" text="models" >}}, and can be added to any smart machine running on Viam.
+You can extend Viam by creating a custom {{< glossary_tooltip term_id="module" text="module" >}} that provides one or more modular {{< glossary_tooltip term_id="resource" text="resources" >}} {{< glossary_tooltip term_id="model" text="models" >}} and can be added to any smart machine running on Viam.
```
Comment (sguequierre) [inline docs/modular-resources/create/_index.md:21]:
```suggestion
You can extend Viam by creating a custom {{< glossary_tooltip term_id="module" text="module" >}} that provides one or more modular {{< glossary_tooltip term_id="resource" text="resource" >}} {{< glossary_tooltip term_id="model" text="models" >}} and can be added to any smart machine running on Viam.
```

File: docs/modular-resources/_index.md (hunk 1 above)
Comment (npentrel) [reply docs/modular-resources/_index.md:72]: Ah. I think the control tab only works for some components and services. But maybe that's not important to call out. How about
```suggestion
   You can test your added resource using the [Control tab](/manage/fleet/#remote-control) and [program](/program/) it with Viam's Go or Python SDKs.
```

**Review by npentrel — APPROVED** [review]

File: docs/modular-resources/_index.md [hunk 3]
```diff
@@ -27,7 +27,7 @@ Viam provides built-in support for a variety of {{< glossary_tooltip term_id="re
 - Various types of hardware {{< glossary_tooltip term_id="component" text="components" >}}.
 - High-level functionality exposed as {{< glossary_tooltip term_id="service" text="services" >}}.
 
-If the component or service you want to use for your project is not natively supported, you can [use _modular resources_ from the Viam Registry](#the-viam-registry) or [create your own modular resources](#modular-resources).
+If the [component](/components/) or [service](/services/) you want to use for your project is not natively supported, you can [use _modular resources_ from the Viam Registry](#the-viam-registry) or [create your own modular resources](#create-your-own-modular-resources).
```
[the remark this answers is not in the document]
Comment (sguequierre) [reply docs/modular-resources/_index.md:30]: Done

**Pull request merged**